### ISSUE TITLE
Support Github App Values

### ISSUE THREAD

**Issue by heruscode** [runatlantis/helm-charts#11]
Hello,

We would like to use this chart with our Github authentication being a Github App instead of a common user.

Would you like a PR for this?

**Comment by Eugst** [comment]
for github App setup I found workaround on a current setup without any changes by providing helm chart values:
```
environmentRaw:
  - name: ATLANTIS_GH_APP_ID
    value: "{{githubApp_id}}"
  - name: ATLANTIS_GH_APP_KEY_FILE
    value: "/var/github-app/key.pem"
  - name: ATLANTIS_WRITE_GIT_CREDS
    value: "true"
environmentSecrets:
  - name: ATLANTIS_GH_WEBHOOK_SECRET
    secretKeyRef:
      name: github-app
      key: webhook-secret

extraManifests:
 - apiVersion: v1
   kind: Secret
   metadata:
    name: github-app
   data:
    key.pem: {{githubApp_pem}}
    webhook-secret: {{githubApp_secret}}

extraVolumes:
  - name: github-app-vol
    secret:
      secretName: github-app
extraVolumeMounts:
  - name: github-app-vol
    mountPath: "/var/github-app"
```

Later(when I will have a bit more spare time)  i will add PR with extending chart values.

**Comment by lkysow** [comment]
Isn't this already supported? https://www.runatlantis.io/docs/access-credentials.html#generating-an-access-token

**Comment by Eugst** [comment]
@lkysow its supported by atlasian app, but helm doesn’t have values to forward it.

**Comment by blacksd** [comment]
@Eugst I went ahead and opened a PR - thanks for the workaround!

**Comment by fishpen0** [comment]
Now that https://github.com/runatlantis/atlantis/pull/1706/files is merged, it seems like the chart can just be updated to support `--gh-app-key` being passed in without all the file nonsense.

**Comment by frank-bee** [comment]
I tried to use the latest chart with githuh app values configured.
At least the github org cannot be configured yet which is required in our case (in the underlying go app there is a parameter for the org)

**Comment by nitrocode** [comment]
Isnt this now already supported by the continuation of closed pr #46 in merged pr #76?
